src/sc.spatialfeature.js

```diff
@@ -12,7 +12,7 @@ export function spatialFeature(storeId, obj) {
     properties: {},
     storeId: storeId,
     style: {},
-    type: 'SPATIAL'
+    type: 'FEATURE'
```

Comment:
We probably want the type to be `Feature` (only the F is capitalized) which is what the spec defines as a valid value.